musica/musica.meta

```diff
@@ -1,22 +1,11 @@
 [ccpp-table-properties]
-  name = micm
+  name = musica
   type = scheme
+  dynamic_constituent_routine = musica_register
```

Comment:
I believe @mattldawson has added the dependencies line in his sandbox, but CAM-SIMA doesn't currently do anything with the dependencies (i.e. grab them during the build)

I've added an agenda item for tuesday's meeting.

Reply:
Thanks for letting me know @peverwhee!  Happy to discuss this Tuesday, although I am guessing if the framework is able to provide the list of dependency files at code generation time it will probably be trivial (e.g. a handful of new lines) to get the correct info into CAM-SIMA for successful building/linking.

Reply:
After our discussion in the CAM-SIMA development meeting, I believe the plan is to wait until the constituents PR for the CCPP-framework is merged in (https://github.com/NCAR/ccpp-framework/pull/549) and then create an issue to update the framework to provide absolute paths for dependencies in the `ccpp_datatable.xml` file. This will allow CAM-SIMA to include the dependencies in the build with minor modification. @nusbaume @peverwhee - is this correct?

Reply:
Yep, that sounds like the correct plan to me!